protocol/0001-MKTF-market_framework.md

```diff
@@ -216,9 +216,24 @@ Market {
 }
 ```
 
+## Successor market
+
+If a market proposal, see [governance](./0028-GOVE-governance.md), designates an existing market as a *parent market* then it must have the same *product*, *settlement asset(s)* and *margin asset(s)*.
+It may propose new risk model and parameters, price monitoring parameters, position and market decimal places and oracles, both for trading terminated and for settlement data.
+Each market can have exactly one market as a *successor* market.
+
+1. if there already is a market (possibly pending i.e. in opening auction, see [lifecycle spec](./0043-MKTL-market_lifecycle.md)) naming a parent market which is referenced in the proposal then the proposal is rejected.
+1. if there are two proposals naming the same parent market then whichever one gets into the pending state first (i.e. passes governance vote) becomes the successor of the named parent; the other proposal is cancelled with reason "parent market not available".
```

Comment:
```suggestion
It may propose a new risk model and parameters, price monitoring parameters, position and market decimal places and oracles, both for trading terminated and for settlement data.
Each market can have exactly one market as a *successor* market.

1. if there already is a market (possibly pending, i.e. in opening auction, see [lifecycle spec](./0043-MKTL-market_lifecycle.md)), naming a parent market, then a subsequent proposal referencing that market is rejected.
1. if there are two proposals naming the same parent market then whichever one gets into the pending state first (i.e. passes governance vote) becomes the successor of the named parent; the other proposal is cancelled with reason "parent market not available".
```

Reply:
"may propose a new risk model and parameters" - Might this be better phrased "may propose new risk model parameters"?

Reply:
I am not sure; the meaning I am trying to convey is that the parameters may be new but also the risk model can be different (and if that happens new parameters are needed because different risk models have different parameter sets).